### PR TITLE
CHANGELOG: release 0.106.0

### PR DESCRIPTION
**Pull request by AnnaShaleva**
T5 states match:
```
anna@kiwi:~/Documents/GitProjects/nspcc-dev/neo-go$ go run scripts/compare-states/compare-states.go http://localhost:20332 http://seed3t5.neo.org:20332
at 0: 62fd8ff9b0543aea352257db5b00bbb01d1bc0d2cc665e1f24cf5de0d16ebc7b vs 62fd8ff9b0543aea352257db5b00bbb01d1bc0d2cc665e1f24cf5de0d16ebc7b
at 4019716: 7258897c12115956488154655db8571568deccb7bd9636c76367946c1d815ce2 vs 7258897c12115956488154655db8571568deccb7bd9636c76367946c1d815ce2
```
Mainnet states match:
```
anna@kiwi:~/Documents/GitProjects/nspcc-dev/neo-go$ go run scripts/compare-states/compare-states.go --ignore-height http://localhost:10335 http://localhost:10332
at 0: 58a5157b7e99eeabf631291f1747ec8eb12ab89461cda888492b17a301de81e8 vs 58a5157b7e99eeabf631291f1747ec8eb12ab89461cda888492b17a301de81e8
at 5417055: 0efff99a79ea5e652abfa4dc361b3b6906ec5aa5da2dac385985e9d6a568b766 vs 0efff99a79ea5e652abfa4dc361b3b6906ec5aa5da2dac385985e9d6a568b766
```
NeoFS testnet containers/netmap are OK:
```
=== RUN   Test_CompareContainers
Height A: 3606303, height B: 3606303
Number of containers checked: 208
Number of netmap entries checked: 4
--- PASS: Test_CompareContainers (0.74s)
PASS
```
NeoFS mainnet containers/netmap are OK:
```
=== RUN   Test_CompareContainers
Height A: 5722968, height B: 5722968
Number of containers checked: 152
Number of netmap entries checked: 5
--- PASS: Test_CompareContainers (0.73s)
PASS
```
Checked against released 3.7.4 version C# node, works OK with both Go RPC and C# RPC:
```
anna@kiwi:~/Documents/GitProjects/nspcc-dev/neo-bench$ make start.MixedFourNodesGoRPC50rate 
./runner.sh --nodes mixed -d "MixedGoRPC4x1" -m rate -q 50 -z 5m -t 30s
make[1]: Entering directory '/home/anna/Documents/GitProjects/nspcc-dev/neo-bench'
=> Stop environment
=> Generate configurations for single-node and four-nodes networks from templates
+ cd ./cmd
+ go run ./config/ --go-template go.protocol.template.yml --go-db leveldb --sharp-template sharp.protocol.template.yml --sharp-db LevelDBStore --msPerBlock 0
creating: 3423809795/go.protocol.template.yml
creating: 3423809795/go.protocol.template.yml
creating: 3423809795/sharp.protocol.template.yml
creating: 3423809795/sharp.protocol.template.yml
make[1]: Leaving directory '/home/anna/Documents/GitProjects/nspcc-dev/neo-bench'
[+] Creating 8/7
 ✔ Network neo_go_network       Created                                                                                                                                                                       0.1s 
 ✔ Container neo-cli-node-two   Created                                                                                                                                                                       0.1s 
 ✔ Container neo_go_node_four   Created                                                                                                                                                                       0.1s 
 ✔ Container neo_go_node_three  Created                                                                                                                                                                       0.1s 
 ✔ Container neo-cli-node-one   Created                                                                                                                                                                       0.1s 
 ✔ Container ir-healthy-1       Created                                                                                                                                                                       0.0s 
 ✔ Container go-node            Created                                                                                                                                                                       0.0s 
 ✔ Container ir-node_healthy-1  Created                                                                                                                                                                       0.0s 
[+] Running 7/7
 ✔ Container neo-cli-node-two   Healthy                                                                                                                                                                       6.0s 
 ✔ Container neo_go_node_four   Healthy                                                                                                                                                                       5.5s 
 ✔ Container neo_go_node_three  Healthy                                                                                                                                                                       5.5s 
 ✔ Container neo-cli-node-one   Healthy                                                                                                                                                                       6.0s 
 ✔ Container ir-healthy-1       Started                                                                                                                                                                       0.2s 
 ✔ Container go-node            Healthy                                                                                                                                                                       5.8s 
 ✔ Container ir-node_healthy-1  Started                                                                                                                                                                       0.2s 
2024/05/21 11:10:04 Used [go-node:20331] rpc addresses
2024/05/21 11:10:04 Run benchmark for MixedGoRPC4x1 :: NEO-GO
2024/05/21 11:10:13 Read 3000000 txs from /dump.txs
2024/05/21 11:10:19 Done 6.128019809s
2024/05/21 11:10:19 Init worker with 50 QPS / 5m0s time limit (3000000 txs will try to send)
2024/05/21 11:10:19 Prepare chain for benchmark
2024/05/21 11:10:19 Determined validators count: 4
2024/05/21 11:10:19 Sending NEO and GAS transfer tx
2024/05/21 11:10:21 Contract hash: ceb508fc02abc2dc27228e21976699047bbbcce0
2024/05/21 11:10:21 Sending contract deploy tx
2024/05/21 11:10:21 Contract was persisted: false
2024/05/21 11:10:22 Contract was persisted: false
2024/05/21 11:10:22 Contract was persisted: false
2024/05/21 11:10:23 Contract was persisted: false
2024/05/21 11:10:23 CPU: 1.372%, Mem: 353.457MB
2024/05/21 11:10:23 Contract was persisted: false
2024/05/21 11:10:24 Contract was persisted: false
2024/05/21 11:10:24 Contract was persisted: false
2024/05/21 11:10:25 Contract was persisted: false
2024/05/21 11:10:25 Contract was persisted: false
2024/05/21 11:10:26 Contract was persisted: false
2024/05/21 11:10:26 Contract was persisted: true
2024/05/21 11:10:26 fetch current block count
2024/05/21 11:10:26 Waiting for an empty block to be processed
2024/05/21 11:10:31 Started test from block = 8 at unix time = 1716289831394
2024/05/21 11:10:32 empty block: 8
2024/05/21 11:10:33 CPU: 5.236%, Mem: 365.492MB
2024/05/21 11:10:36 #9: 230 transactions in 5023 ms - 45.789369 tps
2024/05/21 11:10:41 #10: 255 transactions in 5072 ms - 50.276025 tps
2024/05/21 11:10:43 CPU: 16.855%, Mem: 360.145MB
2024/05/21 11:10:46 #11: 252 transactions in 5045 ms - 49.950446 tps
^C
2024/05/21 11:10:51 received signal: interrupt
2024/05/21 11:10:51 #12: 252 transactions in 5071 ms - 49.694340 tps
2024/05/21 11:10:51 all request workers stopped
2024/05/21 11:10:51 Sent 999 transactions in 20.001635145s
2024/05/21 11:10:51 RPS: 49.946
2024/05/21 11:10:51 All transactions have been sent successfully
2024/05/21 11:10:51 RPC Errors: 0 / 0.000%
2024/05/21 11:10:51 sender worker stopped
2024/05/21 11:10:51 parser worker stopped
2024/05/21 11:10:51 try to write profile
```
```
anna@kiwi:~/Documents/GitProjects/nspcc-dev/neo-bench$ make start.MixedFourNodesSharpRPC50rate 
./runner.sh --nodes mixed --rpc sharp -d "MixedSharpRPC4x1" -m rate -q 50 -z 5m -t 30s
make[1]: Entering directory '/home/anna/Documents/GitProjects/nspcc-dev/neo-bench'
=> Stop environment
=> Generate configurations for single-node and four-nodes networks from templates
+ cd ./cmd
+ go run ./config/ --go-template go.protocol.template.yml --go-db leveldb --sharp-template sharp.protocol.template.yml --sharp-db LevelDBStore --msPerBlock 0
creating: 372126697/go.protocol.template.yml
creating: 372126697/go.protocol.template.yml
creating: 372126697/sharp.protocol.template.yml
creating: 372126697/sharp.protocol.template.yml
make[1]: Leaving directory '/home/anna/Documents/GitProjects/nspcc-dev/neo-bench'
[+] Creating 8/8
 ✔ Network neo_go_network       Created                                                                                                                                                                       0.0s 
 ✔ Container neo-cli-node-one   Created                                                                                                                                                                       0.0s 
 ✔ Container neo-cli-node-two   Created                                                                                                                                                                       0.0s 
 ✔ Container neo_go_node_three  Created                                                                                                                                                                       0.0s 
 ✔ Container neo_go_node_four   Created                                                                                                                                                                       0.0s 
 ✔ Container ir-healthy-1       Created                                                                                                                                                                       0.0s 
 ✔ Container sharp-node         Created                                                                                                                                                                       0.0s 
 ✔ Container ir-node_healthy-1  Created                                                                                                                                                                       0.0s 
[+] Running 7/7
 ✔ Container neo-cli-node-one   Healthy                                                                                                                                                                       6.0s 
 ✔ Container neo-cli-node-two   Healthy                                                                                                                                                                       6.0s 
 ✔ Container neo_go_node_four   Healthy                                                                                                                                                                       5.5s 
 ✔ Container neo_go_node_three  Healthy                                                                                                                                                                       6.0s 
 ✔ Container ir-healthy-1       Started                                                                                                                                                                       0.3s 
 ✔ Container sharp-node         Healthy                                                                                                                                                                       5.9s 
 ✔ Container ir-node_healthy-1  Started                                                                                                                                                                       0.2s 
2024/05/21 11:11:17 Used [sharp-node:20331] rpc addresses
2024/05/21 11:11:17 Run benchmark for MixedSharpRPC4x1 :: Neo_3.7.4_7f227a3026fadeb9723280e6f961b6f29b0f8a7a
2024/05/21 11:11:26 Read 3000000 txs from /dump.txs
2024/05/21 11:11:32 Done 5.979851932s
2024/05/21 11:11:32 Init worker with 50 QPS / 5m0s time limit (3000000 txs will try to send)
2024/05/21 11:11:32 Prepare chain for benchmark
2024/05/21 11:11:32 Determined validators count: 4
2024/05/21 11:11:32 Sending NEO and GAS transfer tx
2024/05/21 11:11:34 Contract hash: ceb508fc02abc2dc27228e21976699047bbbcce0
2024/05/21 11:11:34 Sending contract deploy tx
2024/05/21 11:11:34 Contract was persisted: false
2024/05/21 11:11:34 Contract was persisted: false
2024/05/21 11:11:35 Contract was persisted: false
2024/05/21 11:11:35 Contract was persisted: false
2024/05/21 11:11:36 CPU: 0.881%, Mem: 332.809MB
2024/05/21 11:11:36 Contract was persisted: false
2024/05/21 11:11:36 Contract was persisted: false
2024/05/21 11:11:37 Contract was persisted: false
2024/05/21 11:11:37 Contract was persisted: false
2024/05/21 11:11:38 Contract was persisted: false
2024/05/21 11:11:38 Contract was persisted: false
2024/05/21 11:11:39 Contract was persisted: true
2024/05/21 11:11:39 fetch current block count
2024/05/21 11:11:39 Waiting for an empty block to be processed
2024/05/21 11:11:44 Started test from block = 8 at unix time = 1716289903909
2024/05/21 11:11:44 empty block: 8
2024/05/21 11:11:46 CPU: 4.944%, Mem: 347.449MB
2024/05/21 11:11:49 #9: 230 transactions in 5022 ms - 45.798487 tps
2024/05/21 11:11:54 #10: 254 transactions in 5069 ms - 50.108503 tps
2024/05/21 11:11:56 CPU: 26.764%, Mem: 357.938MB
^C
2024/05/21 11:11:57 received signal: interrupt
2024/05/21 11:11:58 Sent 684 transactions in 13.700417169s
```

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,105 @@
 
 This document outlines major changes between releases.
 
+## 0.106.0 "Zephyrantheses" (21 May 2024)
```
Comment (roman-khimov) [inline CHANGELOG.md:5]: Zephyranthes? The first one I couldn't find in `/usr/share/dict/words`.

Comment (AnnaShaleva) [reply CHANGELOG.md]: Fixed! Also, I'm open to suggestions, may be you have a better name.

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,105 @@
...
+
+We're rolling out a large set of updates including all of Neo 3.7.4 protocol changes:
+native contracts update functionality and extended native contract APIs united under
+the upcoming Cockatrice hardfork. For smart contract developers an ability to
+generate smart contract bindings with dynamic hash is supported as far as the number
```
[the remark this answers is not in the document]
Comment (AnnaShaleva) [reply CHANGELOG.md]: Fixed. Do you think we need to mention something else in the release preview?

[the remark this answers is not in the document]
Comment (roman-khimov) [reply CHANGELOG.md]: It already has a bit too many things to me, so we certainly don't need to add anything.

**Review by roman-khimov — APPROVED** [review]

**Pull request merged**